### PR TITLE
Added count down timer in resend button

### PR DESCRIPTION
**Pull request by kj-crypto**

### CODE REVIEW

File: electrum/gui/qt/email_notification_dialogs.py [hunk 1]
```diff
@@ -248,31 +252,81 @@ def __init__(self, parent, email, error_msg=''):
...
+    def cool_down_task(self):
+        t0 = datetime.datetime.now()
+        while True:
+            self.elapsed_time = (datetime.datetime.now() - t0).total_seconds()
+            left_time = self.get_left_time()
+            self._set_resend_button_status(self.get_formatted_left_time(), False)
+            if left_time < 1:
+                break
+            time.sleep(1)
+        self.on_success()
```
Comment (PJLewandowski) [inline electrum/gui/qt/email_notification_dialogs.py:309]: Is it necessary to call `on_success` here? This function is already assigned to thread.

File: electrum/gui/qt/email_notification_dialogs.py [hunk 2]
```diff
@@ -248,31 +252,81 @@ def __init__(self, parent, email, error_msg=''):
...
+
+    def get_formatted_left_time(self) -> str:
+        return f'{self.get_left_time():>2}s'
+
+    def cool_down_task(self):
+        t0 = datetime.datetime.now()
+        while True:
+            self.elapsed_time = (datetime.datetime.now() - t0).total_seconds()
+            left_time = self.get_left_time()
+            self._set_resend_button_status(self.get_formatted_left_time(), False)
```
Comment (PJLewandowski) [inline electrum/gui/qt/email_notification_dialogs.py:305]: Didnt test, but in this case dont we have an instant switch between labels `0` -> `resend`? Consider moving this between `if` and `sleep`.

File: electrum/gui/qt/email_notification_dialogs.py (hunk 1 above)
Comment (kj-crypto) [reply electrum/gui/qt/email_notification_dialogs.py]: Done.

File: electrum/gui/qt/email_notification_dialogs.py (hunk 2 above)
Comment (kj-crypto) [reply electrum/gui/qt/email_notification_dialogs.py]: Logic was corrected.

**Review by PJLewandowski — APPROVED** [review]

**Pull request merged**